src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/page.tsx

```diff
@@ -153,6 +112,63 @@ export default async function DashboardPage() {
   const yearlySubscriptionUrl = getPremiumSubscriptionUrl({ type: "yearly" });
   const fxaSettingsUrl = process.env.FXA_SETTINGS_URL!;
   const profileStats = await getProfilesStats();
+  let additionalSubplatParams = new URLSearchParams(
+    cookiesList.get("attributionsLastTouch")?.value,
+  );
+
+  // store utm attributions if present in cookies
+  if (cookiesList.get("attributionsFirstTouch")?.value) {
+    const searchParams = new URLSearchParams(
+      cookiesList.get("attributionsFirstTouch")?.value,
```

Comment:
nit: Don't think this needs the `?` in `?.value` since we're already gating above on L120.
```suggestion
      cookiesList.get("attributionsFirstTouch").value,
```

Reply:
Thanks!

Reply:
Reverting actually, got a lint error